stdlib/public/core/StaticString.swift

```diff
@@ -51,16 +95,14 @@ public struct StaticString
   /// - bit 0: set to 0 if `_startPtrOrData` is a pointer, or to 1 if it is a
   ///   Unicode scalar.
   ///
-  /// - bit 1: set to 1 if `_startPtrOrData` is a pointer and string data is
-  ///   ASCII.
+  /// - bit 1: set to 1 if `_startPtrOrData` is an ASCII scalar or sequence.
   @usableFromInline
   internal var _flags: Builtin.Int8
 
-  /// A pointer to the beginning of the string's UTF-8 encoded representation.
+  /// A pointer to a null-terminated sequence of UTF-8 code units.
   ///
-  /// The static string must store a pointer to either ASCII or UTF-8 code
-  /// units. Accessing this property when `hasPointerRepresentation` is
-  /// `false` triggers a runtime error.
+  /// - Warning: Accessing this property when `hasPointerRepresentation` is
```

Comment:
Runtime errors don't rise to the level that we use for `- Warning:` — this and the others like it below should use `- Important:` instead.